src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx

```diff
@@ -155,100 +55,19 @@ export const DateFilter: FC<Props> = ({
         // @ts-expect-error remove line when inputdropdown types fixed in sds
         onClick={handleClick}
       />
-      <Menu
+      <DateFilterMenu
+        {...props}
```

Comment:
I thought the `...props` usually came last so you could override the other props being explicitly set here by passing something from whatever parent component is using `DateFilter`?